pyproject.toml

```diff
@@ -0,0 +1,49 @@
+[build-system]
+requires = ["Cython~=3.0.11", "setuptools>=64", "setuptools-scm>=8"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "fugashi"
+description = "Cython MeCab wrapper for fast, pythonic Japanese tokenization."
+readme = "README.md"
+requires-python = ">=3.8"
+license.text = "MIT"
+authors = [{ name = "Paul O'Leary McCann", email = "polm@dampfkraft.com" }]
+classifiers = [
+  "Environment :: Console",
+  "Intended Audience :: Developers",
+  "Intended Audience :: Science/Research",
+  "License :: OSI Approved :: MIT License",
+  "Natural Language :: Japanese",
+  "Operating System :: POSIX :: Linux",
+  "Operating System :: MacOS :: MacOS X",
+  "Programming Language :: Cython",
+  "Programming Language :: Python :: 3",
+  "Topic :: Text Processing :: Linguistic",
+]
```

Comment:
I used the contents of `setup.cfg`.
Until now, only the contents of `setup.py` were used.